### PR TITLE
Adding dialog editor designs

### PR DESCRIPTION
**Pull request by cshinn**
Adds designs for the dialog editor that bring it in line with other canvas-style pages

### CODE REVIEW

File: UX/ui-service/designer/Dialogs/design.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Dialog Editor
+
+### Blank Dialog Editor
+
+![Blank dialog editor](img/DialogEditor-01.png)
+- When the user creates a new dialog, they are brought to this screen with a single tab and section already populated.
+
+#### Implementation Details
+- The Dialog Editor should follow the same conventions as the blueprint editor when generating names for new dialogs and dialog elements. (e.g. Untitled Dialog")
```
Comment (serenamarie125) [inline UX/ui-service/designer/Dialogs/design.md]: missing quote on Untitled Dialog"

File: UX/ui-service/designer/Dialogs/design.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Clicking the edit icon should bring up the element editor modal
+
+#### Implementation Details
+  - The title of the modal should be "Edit [type of element]"
+
+### Stretch Featues (Optional)
+![Editor with properties panel](img/DialogEditor-06.png)
+- Space and technology permitting, the element editor modal should be replaced with an element properties panel that enables the user to edit the selected element in real time.
+- Undo and Redo features would be required for this to be effective
+- Checking the "Preview Dialog" option would hide the toolbox and properties panel and display the dialog as it would appear when it is being used.
```
Comment (serenamarie125) [inline UX/ui-service/designer/Dialogs/design.md]: @cshinn during our review earlier this week, I think we mentioned that a button would be more appropriate than a checkbox.  Could you make this change?

File: UX/ui-service/designer/Dialogs/design.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+- "Dialogs" is a secondary item under the "Designer" category
+- This area is for the Service Designer and is not visible to other users
+- This area is provided so the Designer can create and edit service dialogs
+
+## Dialog Editor
+
+### Blank Dialog Editor
+
+![Blank dialog editor](img/DialogEditor-01.png)
```
Comment (serenamarie125) [inline UX/ui-service/designer/Dialogs/design.md]: @cshinn Should "Save" be enabled in this mock?
Should we remove dialog details?  I'm not sure we need it for SUI, but I may be wrong :)

File: UX/ui-service/designer/Dialogs/design.md (hunk 2 above)
Comment (beanh66) [inline UX/ui-service/designer/Dialogs/design.md]: @cshinn I agree with @serenamarie125 we should use a button instead. I would put this button up a level next to the save button though. To me, it is a higher level action and changes the view of the canvas to show the preview.

**Comment by beanh66** [comment]
@cshinn LGTM!

**Pull request merged**